mongofiles/mongofiles.go

```diff
@@ -84,7 +94,7 @@ func New(opts Options) (*MongoFiles, error) {
 	}
 
 	if err := mf.ValidateCommand(opts.ParsedArgs); err != nil {
```

Comment:
I agree with you, but since this is intended to be a very minor change (it's not even ticketed), I'd prefer to keep the changes minimal and restricted to just improving the error reporting.